src/styles/block/catalog.scss

```diff
@@ -0,0 +1,11 @@
+.catalog {
+  @include grid-colums-count;
```

Comment:
It looks like there might be a typo in the mixin name 'grid-colums-count'. According to the code style rules, names should be spelled correctly. Please check if it should be 'grid-columns-count' instead.